playbooks/k8s-cluster.yml

```diff
@@ -38,7 +38,7 @@
     kubectl_localhost: true
     kubeconfig_localhost: true
     helm_enabled: true
-    tiller_node_selectors: "node-role.kubernetes.io/master=''"
+    tiller_node_selectors: "node-role.kubernetes.io/node=''"
```

Comment:
Was there a technical reason for this? We've been trying to leave mgmt services running on non-compute nodes

Reply:
In another way the install process fails.